src/paas_charm/charm_state.py

```diff
@@ -209,13 +213,16 @@ class IntegrationsState:
         s3_parameters: S3 parameters.
         saml_parameters: SAML parameters.
         rabbitmq_uri: RabbitMQ uri.
+        tracing_relation_data: The tracing uri provided by the Tempo coordinator charm
+        and charm name.
     """
 
     redis_uri: str | None = None
     databases_uris: dict[str, str] = field(default_factory=dict)
     s3_parameters: "S3Parameters | None" = None
     saml_parameters: "SamlParameters | None" = None
     rabbitmq_uri: str | None = None
+    tracing_relation_data: "TempoParameters | None" = None
```

Comment:
Perhaps we need to move the other class definitions up

Reply:
Moved the classes to the top of file and removed `"`.